### ISSUE TITLE
Feature Request: Verknüpfungen bearbeiten - danach anwenden.

### ISSUE THREAD

**Issue by poidi** [hobbyquaker/homematic-manager#124]
Hallo allseits.

ich würde es noch praktischer finden, wenn man mehrere Einstellungen bearbeiten könnte, und dann alles im Bulk erst anwenden kann.

Also ich erstelle 3 Direktverknüpfungen, und erst mit einem Button "Apply" wird dann alles wirklich auf die Komponenten verteilt. So könnte man Dinge noch bequemer ändern, und müsste nicht immer wieder mal auf die Kommunikation warten.

**Comment by litti** [comment]
Du meinst DV für unterschiedliche Geräte? Denn Du kannst ja mehrere DV auf einmal erstellen (zum Beispiel als Sender einen Taster auswählen und als Empfänger drei "Rollläden").

**Comment by poidi** [comment]
Ja, ich würde gerne alle möglichen Verknüpfungen einstellen, und dann mithilfe von dem Apply, alle veränderten Daten zur Hommatic senden.

**Comment by hobbyquaker** [comment]
Umgesetzt in 3.0.0-beta.0, und zwar genau so, wie du es beschrieben hast: Änderungen an mehreren Verknüpfungen (und an Parametersätzen) lassen sich **sammeln und mit einem „Anwenden" gemeinsam schreiben** [#124].

Ein Review-Dialog zeigt vorher, was genau geschrieben wird; die Übertragung läuft über die getaktete Schreibwarteschlange mit Fortschritt und Abbrechen; ein fehlgeschlagener Eintrag behält seinen Grund und bleibt im Satz, statt alles hinzuwerfen. Dazu passend: mehrere Verknüpfungen lassen sich in einem Zug löschen [#80], und jedes Paar einer Mehrfach-Verknüpfung bekommt eigenen Namen und Beschreibung [#87].

Beta: https://github.com/hobbyquaker/homematic-manager/releases/tag/v3.0.0-beta.0

---
_Dieser Kommentar wurde von Claude, dem KI-Agenten des Maintainers, in dessen Auftrag geschrieben (hobbyquaker hat die Triage der offenen Issues zum 3.0-Beta-Release beauftragt). / This comment was written by Claude, the maintainer's AI agent, on his behalf as part of the issue triage for the 3.0 beta._
